paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -1701,4 +1744,12 @@ class PaparazziPluginTest {
   }
 
   private fun File.registerForDeletionOnExit() = apply { filesToDelete += this }
+
+  companion object {
+    private val CONFIG_ADAPTER = Moshi.Builder().build()!!.adapter(Config::class.java).indent("  ")
```

Comment:
```suggestion
    private val CONFIG_ADAPTER = Moshi.Builder().build()!!.adapter(Config::class.java)
```
this one is not serializing, or is it validating indentation?

Reply:
oops, good catch.  updated.